authors/.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
```

Comment:
why do we have this here?

Reply:
I made a mistake when creating the travis file, it should'nt be in this position. I'll remove it.